docs/usage.md

```diff
@@ -486,6 +494,19 @@ The following tools will produce multi-sample taxon tables:
 
 Note that the multi-sample tables from the 'native' tools in each folders are [not inter-operable](https://taxpasta.readthedocs.io/en/latest/tutorials/getting-started/) with each other as they can have different formats and can contain additional and different data. In this case we refer you to use the standardised and merged output from Taxpasta, as described above.
 
+### Downstream samplesheets
+
+To generate the samplesheet for `genomic-medicine-sweden/metaval`, enable the following parameters:
```

Comment:
Given this needs to be so specific, maybe we should just create a very specific parameter option?

Or at least actually validate this udring pipeline initialisation subworkflow (input parameter check section)